server/java/docker_entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Entrypoint for the tickethub Docker container.
+
+set -e
+set -u
+set -x
```

Comment:
[nit] `set -eux`?

Reply:
That was deliberate, I like it like this for entirely subjective reasons.